test/openscap_parser/test_result_file_test.rb

```diff
@@ -29,6 +29,26 @@ def setup
     end
 
     context 'profiles' do
+      test 'profile_id' do
+        assert_match(/^xccdf_org.ssgproject.content_profile_C2S/,
+                     @test_result_file2.benchmark.profiles.first.id)
+      end
+      test 'profile_selected_rule_ids' do
```

Comment:
Would be nice to see some tests that show that the stuff that should be NOT included is not in the result. It's probably `assert_not_includes` or `assert_not_include`...